cmd/secret/cmd/root.go

```diff
@@ -136,6 +161,8 @@ func (r *root) setCmd() *cobra.Command {
 	setCmd.Flags().StringVarP(&cipherKey, "cipher-key", "c", cipherKey, "cipher key for data encryption and decryption")
 	setCmd.Flags().StringVarP(&path, "path", "p", "file.txt", "the place where the key/value will be stored/got")
 	setCmd.Flags().StringVarP(&redisURL, "redis-url", "r", "", "redis url address. Example: localhost:6379")
+	setCmd.Flags().StringVarP(&postgresURL, "postgres-url", "s", "", "postgres url address. Example: localhost:5432")
+	setCmd.Flags().StringVarP(&migration, "migration", "m", "", "migration up route to scripts/migrations folder. Example: file://../../")
```

Comment:
`migration` should not assume that final migrations lie in some specific `scripts/migrations` folder. It should provide full path to a final folder in specified format. For example, `file://../../scripts/migrations`